packages/rulesets/src/spectral/test/avoid-nested-properties.test.ts

```diff
@@ -64,3 +73,79 @@ test("AvoidNestedProperties should find no errors", () => {
     expect(results.length).toBe(0)
   })
 })
+
+test("AvoidNestedProperties should find errors when the extension is specified outside the complex type definition", () => {
+  const myOpenApiDocument = {
+    swagger: "2.0",
+    info: {
+      description:
+        "Use these APIs to manage Azure Websites resources through the Azure Resource Manager. All task operations conform to the HTTP/1.1 protocol specification and each operation returns an x-ms-request-id header that can be used to obtain information about the request. You must make sure that requests made to these resources are secure. For more information, see https://msdn.microsoft.com/en-us/library/azure/dn790557.aspx.",
+    },
```

Comment:
As its just a test, not sure if we'd need this very descriptive description

Reply:
hmm doesnt hurt to have a meaningful description